src/content/wrapped-ether.md

```diff
@@ -0,0 +1,117 @@
+
+## Wrapped Ether (WETH) vs Ether (ETH): What is the difference? {#weth-vs-eth-differences}
+Although they look similar, Wrapped Ether (WETH) and Ether (ETH) have subtle differences you should be aware of. For example, the Ethereum protocol recognizes Ether—but it has no native knowledge of WETH (and ERC-20 tokens), causing both assets to behave differently. We have listed other important distinctions between ETH and WETH tokens below: 
+
+|            | **Ether (ETH)**                                                                                                                                                                                                                 | **Wrapped Ether (WETH)**                                                                                                                                                                                                                                                                                    |
+|------------|-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| Supply     | **As Ethereum’s native token, the supply of ETH is managed by the  protocol.** The supply of ETH increases when Ethereum validators earn rewards for processing transactions and creating blocks.                           | **WETH is an ERC-20 token whose supply is managed by a smart contract.** New units of WETH are issued by the contract after it receives ETH deposits from users.                                                                                                                                        |
+| Ownership  | **Ownership of ETH is defined and guaranteed by the Ethereum blockchain.** For example, protocol rules ensure you can always spend from an account’s ETH balance if you control the [private key](/glossary/#private-key).  | **Ownership of WETH depends on the WETH token contract.** The contract defines ownership of WETH tokens and determines the conditions under which you can access your WETH balance.                                                                                                                                            |
```

Comment:
```suggestion
| Ownership  | **Ownership of ETH is defined and guaranteed by the Ethereum blockchain.** Protocol rules ensure you can always spend from your ETH if you control the [private key](/glossary/#private-key).  | **Ownership of WETH depends on the WETH token contract.** The contract defines ownership of WETH tokens and determines the conditions under which you can access your WETH balance.                                                                                                                                            |
```

Reply:
what conditions are those for weth? might a common question

Reply:
Tbh, I haven't read the WETH contract code thoroughly, but I assume the most important condition would be that users need to have some nonzero balance in the contract. Since WETH is simple, there'll be no need to check things like whether the contract is paused/active or if a particular user is blacklisted. 

But wouldn't explaining this mean taking readers deep into the weeds? I guess the idea was to keep technical information to a minimum.